src/dynamo/models/comment.js

```diff
@@ -35,6 +35,7 @@ const client = createClient({
     [FIELDS.ID]: 'S',
     [FIELDS.DATE]: 'N',
     [FIELDS.PARENT]: 'S',
+    [FIELDS.OWNER]: 'S',
```

Comment:
When testing creating dynamodb tables the comment table failed with error that one of the secondary indices refers to an attribute that didn't exist. The ByOwner index was added within the last month. At least from my local dynamo contents the owner field exists though the ByOwner index doesn't.

We should check if the ByOwner index exists in production.